src/main/java/org/monarchinitiative/fhir2hpo/loinc/DefaultLoinc2HpoAnnotation.java

```diff
@@ -173,6 +178,30 @@ private MethodConversionResult convertValueQuantity(Observation observation) {
 		return result;
 	}
 
+	private MethodConversionResult convertValueString(Observation observation) {
+		MethodConversionResult result = new MethodConversionResult("ValueString");
```

Comment:
Is there a constant that you could use here for the `MethodConversionResult`?

Reply:
These will probably become an enum at some point, but I didn't want to go too far down that path until we have other types of annotations. Heath brought this up in a previous review, but I don't remember if I logged a TODO or ticket for it. I'll look around.